apps/payment/models.py

```diff
@@ -298,9 +297,32 @@ def save(self, *args, **kwargs):
                 raise NotAcceptable("Insufficient funds")
 
             self.user.save()
-        super(PaymentTransaction, self).save(*args, **kwargs)
+        super().save(*args, **kwargs)
 
     class Meta:
         ordering = ['-datetime']
         verbose_name = _('transaksjon')
         verbose_name_plural = _('transaksjoner')
+
+
+class PaymentReceipt(models.Model):
+    """Transaction receipt"""
+    receipt_id = models.CharField(max_length=128, default=None)
+    to_mail = models.EmailField()
+    from_mail = models.EmailField()
+    subject = models.TextField()
+    description = models.TextField()
+    transaction_date = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
+        if not self.receipt_id:
+            self.receipt_id = str(uuid.uuid4())
+        super().save(*args, **kwargs)
+
+
+class ReceiptItem(models.Model):
```

Comment:
If you connect PaymentReceipt to a Payment, the Payment will likely have its own (sub)items attached, therefore making this model redundant.